Networking/NetworkingTests/Mapper/ProductMapperTests.swift

```diff
@@ -189,6 +190,23 @@ final class ProductMapperTests: XCTestCase {
         XCTAssert(productImage.alt?.isEmpty == true)
     }
 
+    /// Test that product downloadable files are properly mapped.
+    ///
+    func test_that_product_downloadable_files_are_properly_mapped() {
```

Comment:
Can you use our practice Given/When/Then or Arrange/Action/Assert? Look at [this](https://github.com/woocommerce/woocommerce-ios/blob/2b32dcd38531c3bb320fd793a77bec5f85684847/WooCommerce/WooCommerceTests/ViewRelated/inAppFeedback/InAppFeedbackCardViewControllerTests.swift) for an example.

Reply:
Might be worth also comparing the full `ProductDownload` after making it Equatable in case we add more properties to the model.

Reply:
Great suggestion. Changed accordingly